flavors/template-Exasol-all-python-3.10-cuda-conda/flavor_base/language_deps/packages/conda_packages

```diff
@@ -1,13 +1,8 @@
-libxcrypt|4.4.36 # Needed for building the udfclient and because libcrypt got removed from conda sysroot, see https://github.com/conda-forge/linux-sysroot-feedstock/issues/52
 python|3.10.4
 numpy|1.26.2
 pandas|2.2.2
 pyarrow|16.0.0
 libarrow|16.0.0=he216f8a_1_cuda
 libblas|3.9.0=15_linux64_mkl
```

Comment:
The question is do we still need to define this or can conda later decide which to use

Reply:
Removed it. It works